source/agora/node/FullNode.d

```diff
@@ -286,6 +299,7 @@ public class FullNode : API
 
     public override void putTransaction (Transaction tx) @safe
     {
+        endpoint_request_stats.increaseAgoraEndpointCallsTotalBy(1, "transaction", "http");
```

Comment:
It would be very useful to differentiate between received txs and also accepted txs, two different stats.

Also these stats need to be consistently named. For example the API `hasTransactionHash` stat is named `has_transaction_hash`, but this one is named `transaction`. Let's give it the full name `put_transaction`.

Alternatively we could add a prefix `api_put_transaction`, `api_has_transaction_hash`, so it's very easy to find these in the stats.

Reply:
"It would be very useful to differentiate between received txs and also accepted txs, two different stats."
we have those accepted/rejected/received metrics in Transaction.d, this one is only for the endpoint calls

 "Also these stats need to be consistently named. For example the API hasTransactionHash stat is named has_transaction_hash, but this one is named transaction. Let's give it the full name put_transaction."

I named everything based on the name of the http endpoint:
hasTransactionHash function -> has_transaction_hash http endpoint
putTransaction functin -> transaction http endpoint
it would be great if vibe.d would expose the generated http endpoints, so I wouldn't even need to hardcode them

"Alternatively we could add a prefix api_put_transaction, api_has_transaction_hash, so it's very easy to find these in the stats."

we can search for metrics with metric name agora_endpoint_call_total to get all the endpoint(API) metrics